orchagent/qosorch.cpp

```diff
@@ -103,6 +106,14 @@ task_process_status QosMapHandler::processWorkItem(Consumer& consumer)
     SWSS_LOG_ENTER();
 
     sai_object_id_t sai_object = SAI_NULL_OBJECT_ID;
+    /*
+     ToDo:
+     As we are going to have more than one QosMap in type DSCP_TO_TC_MAP, TC_TO_PRIORITY_GROUP_MAP and TC_TO_QUEUE_MAP,
```

Comment:
As discussed offline, we do not need to apply any maps at the switch level. Please update the PR to create separate qos maps for the tables that contain the tunnel mappings